### ISSUE TITLE
Invalid roles are not ignored (IE 11)

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/standards-support#281]
ARIA 1.2 specifies that unknown or abstract ARIA roles should be ignored (https://www.w3.org/TR/wai-aria-1.2/#document-handling_author-errors_roles). This works fine in Chrome and Firefox, but unfortunately not in IE 11. In IE 11, JAWS 2019 only correctly outputs input fields with wrong roles. Other elements with a wrong ARIA role are output differently either without a role or only with their role for certain navigation methods, or depending on the mode at the previous element. 

```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>wrong roles</title>
	</head>
	<body>
		<h1 role="foo">wrong roles</h1>
		<input type=text role="foo" aria-label=search>
		<input type=checkbox role="structure" aria-label=YES>
		<input type=submit role="structure"> 
		<label><select role="structure"><option>1<option>2</select> test</label> 
	</body>
</html> 
```

**Comment by lamador** [comment]
When testing with the latest version of JAWS (2020.2006.12) and IE11 (11.959.18362.0), the "search" `input` element was the only form field detected by JAWS.  Using the virtual PC cursor or the "F" quick nav key skips over the checkbox, submit button and combobox controls and these fields are not listed in the "Select a Form Field" window. Additionally, the "Wrong Roles" heading is not recognized as a heading, but the text can be navigated to using the virtual cursor. When examining the UI Automation tree, the "ControlType" for all the elements on the page is properly exposed. Per the [ARIA 1.2 spec regarding roles](https://www.w3.org/TR/wai-aria-1.2/#document-handling_author-errors_roles), these issues are with the IE11 browser rather than JAWS since "...the user agent MUST treat the element as if no role had been provided."
